numpy/ma/extras.py

```diff
@@ -784,6 +785,7 @@ def replace_masked(s):
     if np.issubdtype(asorted.dtype, np.inexact):
         # avoid inf / x = masked
         s = np.ma.sum([low, high], axis=0, out=out)
+        print(repr(s.data))
```

Comment:
Probably not intended!

Reply:
Nice catch! Must have been in my working copy or stash or something

Reply:
Fixed